examples/image/compute_gray_with_squidpy.im.ImageContainer.apply

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+"""
+Convert to grayscale using :meth:`squidpy.im.ImageContainer.apply`
+--------------------
+This example shows how to use :meth:`squidpy.ImageContainer.apply` to convert an image layer to grayscale.
+This calls :func:`squidpy.im.process` in the background.
+We take the mean of the three channels(RGB) using the defined function 'rgb2gray' to convert the image to grayscale.
+The function 'rgb2gray' calls numpy method 'mean' to calculate the mean(average) of three channels (RGB) across axis 3.
+.. seealso::
+    - :ref:`sphx_glr_auto_examples_image_compute_smooth.py`
+    - :ref:`sphx_glr_auto_examples_image_compute_process_hires.py`
+    - :ref:`sphx_glr_auto_examples_image_compute_gray.py`
+"""
+import squidpy as sq
+
+import numpy as np
+
+import matplotlib.pyplot as plt
+
+###############################################################################
+# First, we load the H&E stained tissue image.
+# Here, we only load a cropped dataset to speed things up.
+# In general, :meth:`squidpy.im.ImageContainer.apply` can also process very large images.
+# (see :ref:`sphx_glr_auto_examples_image_compute_process_hires.py`).
+img = sq.datasets.visium_hne_image_crop()
+
+###############################################################################
+# The method :meth:`squidpy.im.ImageContainer.apply` takes a function as an input.
+# The same function or a mapping of ```{'{library_id}': function}``` takes a numpy.ndarray as input.
```

Comment:
are you sure the ``` renders the dict correctly ?

Reply:
Yes, when we did the zoom meet, it showed proper preview.
Also, using 2` works the same as here:
``{'{library_id}': function}``.